apps/os/backend/integrations/slack/slack-utils.ts

```diff
@@ -0,0 +1,45 @@
+import { WebClient } from "@slack/web-api";
+import { getDb } from "../../db/client.ts";
+import { env } from "../../../env.ts";
+import { logger as console } from "../../tag-logger.ts";
+import { getSlackAccessTokenForEstate } from "../../auth/token-utils.ts";
+
+/**
+ * Send a notification message to iterate's Slack workspace.
+ * This uses the estate ID configured in ITERATE_NOTIFICATION_ESTATE_ID.
+ *
+ * @param message - The message to send (supports Slack markdown)
+ * @param channel - The channel to post to (defaults to "#building")
+ */
+export async function sendNotificationToIterateSlack(
+  message: string,
+  channel = "#building",
+): Promise<void> {
+  const notificationEstateId = env.ITERATE_NOTIFICATION_ESTATE_ID;
+
+  if (!notificationEstateId) {
+    console.warn("ITERATE_NOTIFICATION_ESTATE_ID not configured, skipping notification");
+    return;
+  }
```

Comment:
Seems weird to tie this to "iterate" when really it could easily be a function that takes an estate id and a channel.

If we really want an iterate-specific helper with this exact signature we could do

```ts
export async function sendNotificationToIterateSlack(message: string, channel = "#building") {
  return sendNotificationToEstateSlack({
    message,
    channel,
    estateId: env.ITERATE_NOTIFICATION_ESTATE_ID,
  })
}
```

where the bulk of this function would be replaced by a more broadly-useful sendNotificationToEstateSlack?